### PR TITLE
Feature/7087 statistic data local 7 day incidence/8112 store multiple local statistics

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Storage / caching of multiple local statistics

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8112

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ class CachingHTTPClient: AppConfigurationFetching, StatisticsFetching, LocalStat
 		case dataVerificationError(message: String?)
 	}
 
-	typealias AdministrativeUnit = String
+	typealias groupID = String
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift:15]: Types should be capitalized! To make it perfect I would add a comment what a 'group id' is like:
> Group ID cluster one to n federal states as regional groups.

(Or something more appropriate)

Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift:15]: Maybe you call this `GroupIdentifier` to not be confused with the existing `groupID` variables.

File: src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift [hunk 2]
```diff
@@ -17,7 +17,7 @@ protocol LocalStatisticsFetching {
 	typealias LocalStatisticsCompletionHandler = (Result<LocalStatisticsResponse, Error>) -> Void
 
 	func fetchLocalStatistics(
-		administrativeUnit: String,
+		groupID: String,
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift:20]:
```suggestion
		groupID: GroupIdentifier,
```
…or whatever is defined then. We have the `typealias` let's use it :)

File: src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift [hunk 3]
```diff
@@ -27,8 +27,10 @@ struct LocalStatisticsResponse {
 	let localStatistics: SAP_Internal_Stats_LocalStatistics
 	let eTag: String?
 	let timestamp: Date
+	let groupID: String
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift:30]:
```suggestion
	let groupID: GroupIdentifier
```
typealias!

File: src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift [hunk 4]
```diff
@@ -27,8 +27,10 @@ struct LocalStatisticsResponse {
...
 
-	init(_ localStatistics: SAP_Internal_Stats_LocalStatistics, _ eTag: String? = nil) {
+	init(_ localStatistics: SAP_Internal_Stats_LocalStatistics, _ eTag: String? = nil, _ groupID: String) {
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Services/LocalStatistics/LocalStatisticsProviding.swift:32]:
```suggestion
	init(_ localStatistics: SAP_Internal_Stats_LocalStatistics, _ eTag: String? = nil, _ groupID: GroupIdentifier) {
```
You get the idea…

**Review by whiskey — CHANGES_REQUESTED** [review]
Mostly things about the `typealias`

**Review by 30mar — APPROVED** [review]

**Review by whiskey — APPROVED** [review]

**Pull request merged**